services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/TransferManagerFactory.java

```diff
@@ -68,7 +64,7 @@ private static Supplier<S3AsyncClient> defaultS3AsyncClient() {
         if (crtInClasspath()) {
             return S3AsyncClient::crtCreate;
         }
-        return S3AsyncClient::create;
+        return S3AsyncClient::createWithMultipartEnabled;
```

Comment:
```suggestion
        return () -> S3AsyncClient.builder().multipartEnabled(true).build();
```
Could we avoid creating the new static method `createWithMultipartEnabled ` on the client and just use this instead?

Reply:
Yeah i contemplated using this, just thought it might be neater to create a new one. Sure, i can change it to this, it makes sense to have one lesser static method to have to test